### PR TITLE
NOD: Add various pages

### PR DESCRIPTION
**Pull request by Mottie**
## Description

Adding various pages to Form 10182 - Request a Board Appeal (Notice of Disagreement)

- Update intro page content
- Add Homeless page
- Add Rep choice and name pages
- Add a basic Eligible issues page (WIP)
- Add board review page

Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/22699

## Testing done

Tests to be added later - we need to present content to stakeholders

## Screenshots

<details>
<summary>Intro page</summary>

Intro page content</details>

<details><summary>Homeless</summary>

Screen Shot 2021-04-13 at 4 58 30 PM</details>

<details><summary>Has representative</summary>

Screen Shot 2021-04-13 at 4 59 15 PM</details>

<details><summary>Rep name</summary>

Screen Shot 2021-04-13 at 5 00 06 PM</details>

<details><summary>Issues (WIP)</summary>

Screen Shot 2021-04-13 at 5 01 05 PM</details>

<details><summary>Review option</summary>

Screen Shot 2021-04-13 at 5 01 42 PM</details>

<details><summary>Review & submit</summary>

lreview & submit</details>

## Acceptance criteria
- [x] Content updated on intro page
- [x] Added various pages

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/appeals/10182/containers/IntroductionPage.jsx [hunk 1]
```diff
@@ -12,67 +22,116 @@ class IntroductionPage extends React.Component {
...
+    const { formId, prefillEnabled, savedFormMessages } = formConfig;
+    const sipOptions = {
+      useActionLinks: true,
+      hideUnauthedStartLink: true,
+      formId,
+      prefillEnabled,
+      pageList,
+      messages: savedFormMessages,
+      startText: 'Start the Board Appeal request',
+    };
```
Comment (micahchiang) [inline src/applications/appeals/10182/containers/IntroductionPage.jsx:34]: I like this pattern!

File: src/applications/appeals/10182/content/OptIn.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  </strong>
+);
+
+export const optInDescription = `By checking this box, you’re withdrawing any
+  issue you’ve selected from the legacy appeals process (this is the process for
+  decisions received before February 19, 2019). Instead, the Board will consider
+  your claim for this condition under the new process.`;
+
+export const optInErrorMessage =
+  'Please opt into the new decision review process to proceed';
```
Comment (micahchiang) [inline src/applications/appeals/10182/content/OptIn.jsx:16]: These quotes could be standardized. Is there a reason backticks are used here?

File: src/applications/appeals/10182/content/contestableIssues.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    </legend>
+  );
+
+export const EligibleIssuesDescription = (
+  <>
+    <div>
+      <p className="vads-u-margin-top--0">
+        These issues are in your VA record. If an issue is missing from this
+        list, you can add it by clicking the <strong>Add condition</strong>{' '}
+        button (NOT YET IMPLEMENTED).
```
Comment (micahchiang) [inline src/applications/appeals/10182/content/contestableIssues.jsx:30]: Is the NOT YET IMPLEMENTED intentional?

File: src/applications/appeals/10182/validations.js [hunk 4]
```diff
@@ -1 +1,15 @@
+import { optInErrorMessage } from './content/OptIn';
+
 export const isValidDate = date => date instanceof Date && isFinite(date);
+
+export const requireIssue = (err, fieldData /* , formData */) => {
```
Comment (micahchiang) [inline src/applications/appeals/10182/validations.js:5]: can formData just be removed from the parameters here?

**Review by micahchiang — APPROVED** [review]
lgtm

File: src/applications/appeals/10182/content/OptIn.jsx (hunk 2 above)
Comment (Mottie) [reply src/applications/appeals/10182/content/OptIn.jsx:16]: I just like to wrap my text so it's readable without making my editor soft-wrap

File: src/applications/appeals/10182/content/contestableIssues.jsx (hunk 3 above)
Comment (Mottie) [reply src/applications/appeals/10182/content/contestableIssues.jsx:30]: Yes it is.... we have to present this to DEPO on Friday and the design is not finalized.

File: src/applications/appeals/10182/validations.js (hunk 4 above)
Comment (Mottie) [reply src/applications/appeals/10182/validations.js:5]: To do 😉  - thanks!

File: src/applications/appeals/10182/containers/IntroductionPage.jsx (hunk 1 above)
Comment (Mottie) [reply src/applications/appeals/10182/containers/IntroductionPage.jsx:34]: `useActionLinks` may not be necessary if the decision to make the save-in-progress start into an action link - see https://github.com/department-of-veterans-affairs/vets-website/pull/16752

**Review by alexandec — APPROVED** [review]

**Pull request merged**